shared/actions/chat/index.js

```diff
@@ -96,7 +96,9 @@ function * _incomingMessage (action: Constants.IncomingMessage): SagaGenerator<a
           return
         }
 
-        yield call(Inbox.processConversation, incomingMessage.conv)
+        if (incomingMessage.conv) {
```

Comment:
It can be null:
```
record IncomingMessage {
    MessageUnboxed message;
    ConversationID convID;
    union { null, ConversationLocal } conv;
    union { null, Pagination } pagination;
  }
```